agent/php_user_instrument.c

```diff
@@ -435,6 +552,10 @@ void nr_php_destroy_user_wrap_records(void) {
  */
 nruserfn_t* nr_wrapped_user_functions = 0;
 
+/*
+ * nr_php_user_function_add_declared_callback is ONLY called from drupal for
+ * PHP < 7.3.  It does not need to be adjusted for OAPI.
+ */
```

Comment:
Theoretically yes, but we still need to determine how we will handle changing the drupal header args for OAPI so we can leave this functionality in now, pending that PR.  If needed it can be changed in that PR as the change is not really relevant to this PR.  I'll add it as a note to that ticket.